docs/src/dev-guide/building-package.md

```diff
@@ -61,6 +62,14 @@ where `<flavour>` is `json` or `text`.
 The tar will be written to `build/clp-<flavour>-<os>-<arch>-v<version>.tar.gz`, with appropriate
 values for the fields in angle brackets.
 
+## Test
+
+To test the package, run:
+
+```shell
+task package-test
+```
+
```

Comment:
Since this PR won't have any package integration tests, I think can we drop this for now. If you like, you could add a testing section to the end of https://docs.yscope.com/clp/main/dev-guide/components-core/index.html